app/src/main/java/com/realityexpander/tasky/presentation/agenda_screen/AgendaScreen.kt

```diff
@@ -88,6 +89,11 @@ fun AgendaScreenContent(
         }
     }
 
+    // Guard against invalid authentication state
+    if (TaskyApplication.authInfoGlobal?.authToken == null) {
+        navigateToLogin()
+    }
```

Comment:
Im still unclear when to use a `SideEffect` handler. 

Is there a resource you could recommend about when to use this? What situation are we trying to avoid?

Reply:
I will do more research on `SideEffects`.